cmd/build.go

```diff
@@ -17,6 +17,11 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+type buildOptions struct {
+	*settings.Config
```

Comment:
I always prefer to use a field (ex: `config *settings.Config`) over an embedded struct. In some cases it doesn't make a difference, but when accessing fields I find it is easier to understand where values come from when it is a field. 

I generally will only use embedded structs as a way to satisfy interfaces.